tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"target": "es6",
+		"lib": ["es6"],
+		"module": "commonJS",
+		"rootDir": "src",
+		// "moduleResolution": "NodeNext",
```

Comment:
It's not a good practice to leave such commented strings. Better to remove it.

Reply:
It was generated by default, I`ll remove commented strings